.github/workflows/github_release.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+base_dir="$( cd "$(dirname "$0")/../.." >/dev/null 2>&1 ; pwd -P )"
+readonly base_dir
+readonly pom_file="$base_dir/parent/pom.xml"
+
+# Read project version from pom file
+project_version=$(grep "<revision>" "$pom_file" | sed --regexp-extended 's/<.{0,1}revision>//g' | xargs)
+readonly project_version
+echo "Read project version '$project_version' from $pom_file"
+
+# Calculate checksum
+readonly artifact="$base_dir/product/target/openfasttrace-${project_version}.jar"
+echo "Calculate sha256sum for file '$artifact'"
+file_dir="$(dirname "$artifact")"
+readonly file_dir
+file_name=$(basename "$artifact")
+readonly file_name
+pushd "$file_dir"
+readonly checksum_file_name="${file_name}.sha256"
```

Comment:
Why the dir magic? Shouldn't a full path relative to `base_dir` do the trick?

Reply:
I simplified the code